### PR TITLE
Better error message for image resizing

### PR DESCRIPTION
**Pull request by zetok**
Now user will know what is the acceptable file size limit, and if
they don't feel satisfied, they can always report a bug.

### CODE REVIEW

File: src/widget/form/profileform.cpp [hunk 1]
```diff
@@ -242,7 +242,8 @@ void ProfileForm::onAvatarClicked()
     // If this happens, you're really doing it on purpose.
     if (bytes.size() > 65535)
     {
-        QMessageBox::critical(this, tr("Error"), tr("This image is too big"));
+        QMessageBox::critical(this, tr("Error"),
+            tr("This image is too big.\nPlease supply image smaller than 64 KiB, or try with different one.\nIf you think that your image should work, please report a bug with it attached."));
```
Comment (tux3) [inline src/widget/form/profileform.cpp]: Some english mistakes as far as I can tell:
Please supply image -> Please supply an image
try with different one -> try with a different one

I'm not sure if "If you think that your image should work, please report a bug with it attached." should be in the error message in this case.
This error will probably never happen, so no need to handheld the user here. I'm not convinced that it's technically possible at all to craft a file that would trigger this error. It's probably impossible. If it happens, then you really tried very hard to make it happen and you can find the bugtracker by yourself :)

**Comment by zetok** [comment]
Thanks, fixed mistakes.

I'm too not sure about last sentence, but.. I have no knowledge whether there won't be some legit image that could trigger this behaviour, and because of that worst-case scenario should IMHO be assumed - i.e. there **_will**_ be people who will run into that issue. Thus for a really rare edge case user should be encouraged to report a bug, along with attaching an image, so that edge-case problem could be fixed, rather than letting user think that "_qTox is broken, since it can't even use my picture_".

**Comment by tux3** [comment]
> I have no knowledge whether there won't be some legit image that could trigger this behaviour

There will not. No legit image could possibly trigger this.

I took the most non-legit image I could, a colorful JPG photo that I tried to make it as bloated and poorly encoded and bloated as I could, with extra pointless metadata and all compression off, and I ended up with this:

ss039

It's 4.2kB and I really tried to make it big. The limit is 64kB, 15 times bigger.

If it happens, this isn't a legit image or a mistake, this is someone with way too much free time.
I don't know if it's possible, maybe it is, but it will be someone who knows exactly what they are doing.

**Comment by zetok** [comment]
If there is consensus that it's not needed, then I'll remove it from PR, ok?

(from my own testing: I wasn't able to trigger this condition, but I haven't tried with many files since it's bothersome to set avatar again and again)

**Comment by Zer0-One** [comment]
IMO you can trim this down to: "The supplied image is too large. Please use an image less than 64KB in size."

**Comment by zetok** [comment]
updated ~according to @Zer0-One's suggestion

**Pull request merged**